### DISCUSSION TITLE
Unable launch chrome due to unknown operator ?? in browser.cjs

### DISCUSSION THREAD

**Discussion by ty-huynh** [spatie/browsershot#786] [Bugs]
Unknown Operator ?? in  browser.cjs file

```
if (request.options.waitForSelector) {
            await page.waitForSelector(request.options.waitForSelector, request.options.waitForSelectorOptions ?? undefined);
        }
```

Should be 

```
if (request.options.waitForSelector) {
            await page.waitForSelector(request.options.waitForSelector, request.options.waitForSelectorOptions ? request.options.waitForSelectorOptions : undefined);
        }
```

Please fix in new version. 

Thanks

**Comment by owuraku** [comment]
Is there any news on this fix?

**Comment by ty-huynh** [comment]
@owuraku 

Seem still not fix yet! 

You can fork your version and make fix at this line: 

https://github.com/spatie/browsershot/blob/main/bin/browser.cjs#L373

Hope this help!

**Comment by bessone** [comment]
I had the same problem on an Ubuntu 20.04 machine running the old, and deprecated, Node 10.x

By updating to Node 16.x the problem is solved without changing the Browsershot code

**Reply by ty-huynh** [reply]
Great found. 

For upgrading nodejs on my current environment may be painful due to a lot of issue with js compile.

**Comment by ty-huynh** [comment]
It not a BUG of library. It's environment software (nodejs version issue) as @bessone comment.
